docs: add how to index unindexed opensearch fields (#933)
elastisys/compliantkubernetes: docs/user-guide/logs.md

![Dashboard](../img/dashboard.png)

> ### Handling Unindexed Fields

Reviewer: ```suggestion
### Refresh field list for index pattern
```

> OpenSearch fields not purposely unindexed can sometimes show up as unindexed, meaning you won't be able to filter or search for those fields in OpenSearch. This can be solved by refreshing the index pattern field list associated with the unindexed fields.

Reviewer: ```suggestion
When new fields are indexed in OpenSearch they are not immediately available for query in OpenSearch Dashboards.
If you find a field missing from a selected index pattern, you can try to to refresh the field list for the particular index pattern.
```

Author: The problem for me with this change is that i have experienced instances where the fields are indexed and properly searchable but then suddenly these fields can't be sarched anymore it doesn't only happen to new fields.

Reviewer: Sounds like a bug more than anything. But feel free to modify as you want

Author: maybe :sweat_smile: and true I can work from these lines and modify them to fit more.

> 
> To refresh the field list do this:
> 

Reviewer: ```suggestion
```

> OpenSearch fields not purposely unindexed can sometimes show up as unindexed, meaning you won't be able to filter or search for those field in OpenSearch. This can be solved by refreshing the index pattern field list associated with the unindexed fields.
> 
> To refresh the field list do this:
> 
> 1. Open the sidebar and click **Dashboard Management** under Management.

Reviewer 2: ```suggestion
OpenSearch fields not purposely unindexed can sometimes show up as unindexed, meaning you won't be able to filter or search for those fields in OpenSearch. This can be solved by refreshing the index pattern field list associated with the unindexed fields.

To refresh the field list do this:

1. Open the sidebar and click **Dashboards Management** under Management.
```

Author: fixed, thanks for finding this :+1:

1. Click **Index patterns** in the top left side.

> 1. Click the index pattern you want to refresh e.g. **kubernetes***.<!-- markdownlint-disable-line MD044 -->

Reviewer 2: Is there 1 extra * here or is this correct?

Author: its correct, the first two on either side makes the word appear in bold while the third one is because the index pattern name is `kubernetes*`
